spec/services/send_reject_by_default_email_to_provider_spec.rb

```diff
@@ -22,4 +22,19 @@
     expect(ActionMailer::Base.deliveries.count).to eq(1)
     expect(training_provider_email['rails-mail-template'].value).to eq('application_rejected_by_default')
   end
+
+  context 'with continuous applications feature flag active' do
```

Comment:
we also have a spec helper for CA tests 🙌 
`spec/requests/candidate_interface/continuous_applications_spec.rb`

Reply:
it does!